src/types/reviewRequest.ts

```diff
@@ -32,10 +34,16 @@ export interface ReviewRequestInfo {
   reviewers: User[]
 }
 
-export interface ReviewRequestDto extends Omit<ReviewRequestInfo, "reviewers"> {
+export interface CreateReviewRequestDto
+  extends Omit<ReviewRequestInfo, "reviewers"> {
   reviewers: string[]
 }
 
+export type UpdateReviewRequestDto = SetOptional<
```

Comment:
hmm quick clarification: why is `reviewers` field set to `optional`?

Reply:
made `reviewers` a required property! thanks for pointing this out!